crates/ide/src/ty/known.rs

```diff
@@ -111,12 +111,16 @@ pub fn flake(inputs: &[&str]) -> Ty {
             },
             "checks": {
                 _: {
-                    _: derivation
+                    _: {
+                        _: derivation
+                    }
```

Comment:
This should be only 2-levels.
`checks.<system>.<name> = <derivation>;`

To avoid confusion, we should document the meaning of each `_`, by either comments or a new `ty!` syntax. I don't have any good idea about the syntax though.

Reply:
fixed the issue and added some comments